x/shared/types/params.go

```diff
@@ -206,3 +247,34 @@ func validateGracePeriodOffsetBlocksIsLessThanNumBlocksPerSession(params *Params
 	}
 	return nil
 }
+
+// validateSupplierUnbondingPeriodIsGreaterThanCumulatedProofWindowCloseBlocks
+// validates that the SupplierUnbondingPeriod blocks is greater than the cumulated
+// proof window close blocks.
+// It ensures that a supplier cannot unbond before the pending claims are settled.
+func validateSupplierUnbondingPeriodIsGreaterThanCumulatedProofWindowCloseBlocks(params *Params) error {
```

Comment:
1. In general, we should use `Cumulative` instead of `Cumulated` in scenarios like this.
2. I suggested alternative naming (PTAL). We'd have to rename everything else that's related.

Reply:
1. Replaced all occurences.
2. Can't find the suggested alternative name :thinking: